runtime/List.res

```diff
@@ -61,8 +61,6 @@
 
 @@config({flags: ["-bs-noassertfalse"]})
 
-type t<'a> = list<'a>
```

Comment:
I think it is good to directly use `list<'a>` in the function signatures, but we should still keep type `t` around for each module as a convention (and to make e.g. usage in functors easier).

OCaml does it this way, too, e.g. https://ocaml.org/manual/5.3/api/String.html - there is a `type t = string`, but the function signatures use `string` directly.

There was some previous discussion on that, but I currently can't find it.

Reply:
so we should add `t` to other modules like Array, DIct, etc?

Reply:
IMHO yes